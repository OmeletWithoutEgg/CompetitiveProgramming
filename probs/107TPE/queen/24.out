131732.46 -3449333.59
