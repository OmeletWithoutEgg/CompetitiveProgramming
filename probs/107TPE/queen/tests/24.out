-549068.00 20182938.00
